src/Toolkit.Forms/Esri.ArcGISRuntime.Toolkit.Xamarin.Forms.csproj

```diff
@@ -69,9 +70,6 @@
     <Compile Include="..\Toolkit\Toolkit\WeakEventListener.cs" Link="WeakEventListener.cs" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(Nullable)'=='annotations'">
-    <Compile Include="..\Toolkit\NullableAttributes.cs" />
-  </ItemGroup>
```

Comment:
I think this was removed because it is included by the main Toolkit project through the reference to `CommonProperties.targets`. The internal attributes are visible to the Forms projects because the Toolkit project has `InternalsVisibleTo` set. 

![image](https://user-images.githubusercontent.com/29742178/153083061-ec5f9d55-839e-4d9c-9792-97634866efde.png)

I can revert this change if it would be better to be explicit and accept this warning.